base/commands/viridian/tokens.go

```diff
@@ -0,0 +1,183 @@
+package viridian
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"sort"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/hazelcast/hazelcast-commandline-client/clc/paths"
+	"github.com/hazelcast/hazelcast-commandline-client/clc/secrets"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/plug"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/viridian"
+)
+
+type Tokens struct {
+	Key          string
+	Token        string
+	RefreshToken string
+	ExpiresIn    int
+}
```

Comment:
How about:
```go
type APICredentials struct {
    Key string
    AccessToken string
    RefreshToken string
    ExpiresIn int
}
```
```

Reply:
Fixed in: https://github.com/hazelcast/hazelcast-commandline-client/pull/300/commits/d3d3b8eb6da57e1de512a87e7ebd5597a1c31c6f